src/tests/asyncTests.ts

```diff
@@ -533,4 +545,34 @@ export class AsyncTests {
       }
     };
   }
+
+  private async describeSObjects(sObjectName: string): Promise<Field[]> {
+    try {
+      const describeRequest =
+        await this.connection.tooling.describe(sObjectName);
+      return describeRequest.fields;
+    } catch (error) {
+      console.error('Error describing sObject:', error);
```

Comment:
Remove console.error. If you want to log the error use the logger. Search code for examples.